index.md

```diff
@@ -1167,60 +1176,70 @@ country = "England"
 s = "{} who is {} lives in {}".format(name, age, country)
 ```
 
+This error is similar to [E1121](#E1121).
+
+
 
 ### Too few format args (E1306) {#E1306}
 
 This error occurs when you use the `format` method on a string, but call it
 with fewer arguments than the number of `{}` in the string.
-This error is similar to E1120, meanwhile opposite to E1305.
 
 ~~~~ {include="E1306_too_few_format_args"}
 ~~~~
 
-Corrected version
+Corrected version:
 
 ```python
 s = "{} and {}".format("first", "second")
 ```
 
+This error is similar to [E1120](#E1120).
 
-### Bad str strip call (E1310) {#E1310}
 
-This error occurs when you call `strip`, `lstrip`, or `rstrip`, but give it
-an argument string which contains duplicate characters.
+### Missing format argument key (W1303) {#W1303}
 
-The argument string should contain the *distinct* characters that you want to
-remove from the end(s) of a string.
+This error occurs when a format string that uses named fields does not receive required keywords. In the following example, we should assign three values for `last_name`, `first_name` and `age`.
```

Comment:
Also I tend to prefer using the Oxford comma (last part of the second sentence).